ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -194,8 +194,8 @@ Verify User Can Edit A Data Science Project
     Should Be Equal As Strings  ${ns_name}  ${ns_newname}
 
 
-Verify User Can Create And Start A Workbench With Ephimeral Storage
-    [Tags]    Sanity    Tier1    ODS-1812
+Verify User Can Create And Start A Workbench With Ephemeral Storage
+    [Tags]    Smoke    ODS-1812
```

Comment:
please remove this from smoke because it gets skipped from RHODS 1.20 on. We can remove the test from the suite at all

Reply:
let's add ODS-1814 instead